src/server/game/Autobroadcast/AutobroadcastMgr.cpp

```diff
@@ -167,7 +167,7 @@ void AutobroadcastMgr::SendWorldAnnouncement(uint8 textId)
         // Get player's locale
         LocaleConstant locale = player->GetSession()->GetSessionDbLocaleIndex();
 
-        if (!_autobroadcasts.empty())
```

Comment:
/facepalm 😄 this basically cancels this method if there are any autobroadcasts ...

Reply:
Not sure what you mean, autobradcast is working?

Reply:
It only works for Notify, not for regular announcement.

this code here 
“if !autobroadcasts.empty
    return”

translate to “if autobroadcasts is not empty, then stop”

So if there are any autobroadcasts, this method is not allowed to continue.